src/hooks/useHashParams.ts

```diff
@@ -0,0 +1,14 @@
+export default function () {
+  const url = window.location.href
```

Comment:
Why don't we use `react-router` here?

Reply:
I didn't use it because hooks from `react-router` can be used only in the render function. So to not overcomplicated our component I created a custom hook that returns an object and not a string